practice/django_practice/employees/models.py

```diff
@@ -0,0 +1,111 @@
+from django.db import models
+from datetime import date
+
+from departments.models import Department
+from django_practice.models import BaseModel
+from django_practice.constants.enum import Gender, ActiveStatus
+from .utils.number_of_age import calculate_age
+from .validate.validation_employee import validate_age, validate_email
+
+
+class EmployeeFilterManager(models.Manager):
```

Comment:
Imagine this is a custom manager for employee model (I mean many method in this class), so the name you gave is really reasonable,
Besides, I don't see any updates about custom QuerySet in this PR.